### PR TITLE
Update Password Managers page to cardv2

### PR DESCRIPTION
**Pull request by nitrohorse**
## Description

This PR:
- updates the Password Managers page to use cardv2
- replaces [KeePass](https://sourceforge.net/projects/passwordsafe/files/) with KeePassXC because
  1. it's Windows-only
  2. XC is cross-platform
  3. XC is more actively maintained.

https://deploy-preview-1170--privacytools-io.netlify.com/software/passwords/#pw

Resolves: N/A

#### Check List 

- [x] I have read and understand [CONTRIBUTING.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] I have listed the source code for this project in [source_code.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/source_code.md).

- [x] This project is [free/libre software](https://www.wikipedia.org/wiki/Free_software).

- ~~[ ] This project has an [associated discussion](https://github.com/privacytoolsIO/privacytools.io/issues).~~

Code Repository (if applicable): N/A

### CODE REVIEW

File: _includes/sections/password-managers.html [hunk 1]
```diff
@@ -1,52 +1,70 @@
...
+  edge="https://www.microsoft.com/store/apps/9P6KXL0SVNNL"
+  android="https://play.google.com/store/apps/details?id=com.x8bit.bitwarden"
+  ios="https://itunes.apple.com/app/bitwarden-free-password-manager/id1137397744?mt=8"
+%}
+
+{%
+  include cardv2.html
+  title="KeePassXC - Local"
+  image="/assets/img/tools/keepassxc.png"
+  description="<strong>KeePass</strong> is a community fork of KeePassXC, a native cross-platform port of KeePass Password Safe, with the goal to extend and improve it with new features and bugfixes to provide a feature-rich, fully cross-platform and modern open-source password manager."
```
Comment (nitrohorse) [inline _includes/sections/password-managers.html]: Typos:
- KeePass -> KeePassXC
- KeePassXC -> KeePassX

File: _includes/sections/password-managers.html [hunk 2]
```diff
@@ -1,52 +1,70 @@
...
-  website="keepass.info"
-  footer="OS: Windows, macOS, Linux, iOS, Android, BSD."
-  description='KeePass is a free open source password manager, which helps you to manage your passwords in a secure way. All passwords in one database, which is locked with one
-  master key or a key file. The databases are encrypted using the best and most secure encryption algorithms currently known: AES and Twofish. See also: <a href="https://keepassxc.org/">KeePassXC</a> with official native cross-platform support for Windows/macOS/Linux.'
-  %}
-  {% include card.html color="warning"
+  description="<strong>Bitwarden</strong> is a free and open source password manager. It aims to solve password management problems for individuals, teams, and business organizations. Bitwarden is among the easiest and safest solutions to store all of your logins and passwords while conveniently keeping them synced between all of your devices. If you don't want to use the Bitwarden cloud, you can easily host your own Bitwarden server."
+  website="https://bitwarden.com/"
+  forum="https://forum.privacytools.io/t/discussion-bitwarden/1343"
+  github="https://github.com/bitwarden"
```
Comment (nitrohorse) [inline _includes/sections/password-managers.html]: @JonahAragon I realize the order of the cardv2 arguments isn't strictly enforced; should I stick with a certain order though?

Comment (nitrohorse) [reply _includes/sections/password-managers.html]: I'm also thinking we the `windows` argument should even be added (although it's supported) to discourage Windows use..

Comment (blacklight447-ptio) [reply _includes/sections/password-managers.html]: Yeah, that would be a smart idea.

Comment (nitrohorse) [reply _includes/sections/password-managers.html]: Okay, will remove :+1:

**Pull request merged**